sdk/core/azure-core/test/ut/transport_adapter_implementation.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "transport_adapter_base.hpp"
+#include <azure/core/context.hpp>
+#include <azure/core/http/policy.hpp>
+#include <azure/core/response.hpp>
+
+#include <iostream>
+#include <string>
+#include <thread>
+
+#include <http/curl/curl_connection_pool_private.hpp>
+#include <http/curl/curl_connection_private.hpp>
+
+using testing::ValuesIn;
+
+namespace Azure { namespace Core { namespace Test {
+
+  /**********************   Define the parameters for the base test and a suffix  ***************/
```

Comment:
I am confused, is it prefix now or still suffix? We still have the `GetSuffix` method.
```suggestion
  /**********************   Define the parameters for the base test and a prefix  ***************/
```

Reply:
suffix!  I have fixed it